### PR TITLE
excluded subclass for MONDO:0001752

### LINKED ISSUE #4165: [Revise subclass]
**Mondo term (ID and Label)**

rdfs:label "alveolar periostitis"^^xsd:string
id "MONDO:0001752"^^xsd:string

**Suggested revision and reasons**

This is currently classified under 

rdfs:label "extrinsic allergic alveolitis"^^xsd:string
id "MONDO:0004553"^^xsd:string
definition "An inflammatory interstitial lung disease caused by hypersensitivity reaction to inhalation or ingestion of antigens. The antigens are usually related to the patient's occupation. It can present as an acute illness with flu-like symptoms, subacute with repeated episodes of pneumonia, or chronic with dyspnea and productive cough. The majority of patients recover following the cessation of the exposure to the antigen that causes the disease. Chronic exposure may eventually progress to interstitial lung fibrosis."^^xsd:string

However, from the definition it looks like it is not a lung disease at all

rdfs:label "alveolar periostitis"^^xsd:string
id "MONDO:0001752"^^xsd:string
definition "A condition sometimes occurring after tooth extraction, particularly after traumatic extraction, resulting in a dry appearance of the exposed bone in the socket, due to disintegration or loss of the blood clot. It is basically a focal osteomyelitis without suppuration and is accompanied by severe pain (alveolalgia) and foul odor. (Dorland, 28th ed)"^^xsd:string

Missclassification is likely due to confusion between the terms alveolalgia (pain) and alveolar (pertaining to the alveoli of the lung).

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

### PR DESCRIPTION
**Pull request by sabrinatoro**
Fixes #4165

### CODE REVIEW

File: src/ontology/mondo-edit.properties [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Tue Jan 04 15:01:29 PST 2022
```
Comment (nicolevasilevsky) [inline src/ontology/mondo-edit.properties:1]: need to remove this change

**Review by nicolevasilevsky — APPROVED** [review]
could you please remove the changes to the properties file, then this can be merged. thanks!

File: src/ontology/mondo-edit.properties (hunk 1 above)
Comment (sabrinatoro) [reply src/ontology/mondo-edit.properties]: i think I removed it

**Comment by sabrinatoro** [comment]
@nicolevasilevsky 
I think I removed the extra file. Can you double-check that you do not see the extra file, please? Thank you

**Comment by nicolevasilevsky** [comment]
yep, looks good! I'll merge

**Pull request merged**